server/lib/search.js

```diff
@@ -218,7 +214,8 @@ export const searchCollectivesInDB = async (
   }
 
   if (tags?.length) {
-    dynamicConditions += `AND c."tags" @> (:searchedTags) `;
+    searchedTags = tags.map(tag => tag.toLowerCase());
+    dynamicConditions += `AND c."tags" @> Array[:searchedTags]::varchar[] `;
```

Comment:
Fixed the condition here to get rid of `searchedTags = `{${tags.map(tag => `"${tag}"`).join(',')}}`;` which was breaking the query when searching for something with `"`.